### DISCUSSION TITLE
What is the suggested configuration for Production?

### DISCUSSION THREAD

**Discussion by xs2tarunkukreja** [vesoft-inc/nebula#5642] [Q&A] [answered]
We are deploying Nebula Graph DB in K8S Cluster. What is the suggested configurations for Nodes and Pods?

Size may be round 1 TB, 10 TB, up to 2 PB

**Comment by xs2tarunkukreja** [comment]
@wey-gu Please suggest.

**Comment by xs2tarunkukreja** [comment]
@wey-gu - Do you have any suggestion or any reference?

**Comment by wey-gu** [comment, marked as answer]
Dear @xs2tarunkukreja

We could refer to https://docs.nebula-graph.io/3.5.0/4.deployment-and-installation/1.resource-preparations/#requirements_and_suggestions_for_installing_nebulagraph_in_production_environments

**Reply by xs2tarunkukreja** [reply]
Thanks
